sample/applicationinsights-react-sample/src/ApplicationInsightsService.tsx

```diff
@@ -19,6 +19,12 @@ appInsights.loadAppInsights();
 appInsights.addTelemetryInitializer((env:ITelemetryItem) => {
     env.tags = env.tags || [];
     env.tags["ai.cloud.role"] = "testTag";
+    //custom props
+    if (env.data) {
```

Comment:
Just a nit, but this could also "add" the env.data if it's not present with something like
```ts
let data = env.data = env.data || {};
```

So then the custom properties are always added.